app/views/pages/netlify_config.yaml.erb

```diff
@@ -26,3 +26,52 @@ collections:
         value_field: "{{slug}}"
         display_fields: ["title", "{{slug}}"]
       - {label: "Body", name: "body", widget: "markdown"}
+      - name: footer
+        label: Footer navigation
+        widget: object
+        collapsed: true
+        fields:
+          - name: visiblility
+            widget: select
+            options:
+              - { label: "Show", value: "show" }
+              - { label: "Hide", value: "hide" }
+          - name: order
+            widget: number
+            hint: "Items with lower numbers are sorted before items with higher numbers."
+            default: "100"
+
+  - name: settings
+    label: Site settings
+    editor:
+      preview: false
+
+    files:
+      - label: "Site footer"
+        name: "footer"
+        description: "Configure the footer at the bottom of every page."
+        file: "_settings/footer.yml"
+        fields:
+          - name: agency_name
+            label: Agency name
+            widget: string
+            required: false
+          - name: agency_phone
+            label: "Phone"
+            required: false
+            hint: "What number can people call with general questions?"
+          - name: agency_email
+            required: false
+          <% (1..5).each do |num| %>
```

Comment:
[suggestion] I think the `list` widget type would be a better match than hard-coding these fields. https://www.netlifycms.org/docs/widgets/#list

Reply:
Yes, I think you are right. Somehow my eyes glided right over it...